jade/release-build.yaml

```diff
@@ -2,9 +2,9 @@
 # ROS buildfarm release-build file
 ---
 abi_incompatibility_assumed: true
-jenkins_binary_job_priority: 82
+jenkins_binary_job_priority: 122
```

Comment:
Well, you get the idea I guess, this was just an example. Here's the updated list with the source jobs with higher priority than the binary ones:

kinetic/release-build.yaml:
jenkins_binary_job_priority: 51
jenkins_source_job_priority: 50

jade/release-build.yaml:
jenkins_binary_job_priority: 61
jenkins_source_job_priority: 60

indigo/release-build.yaml:
jenkins_binary_job_priority: 71
jenkins_source_job_priority: 70